src/pages/movie-details/index.html

```diff
@@ -0,0 +1,66 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  
+  <script type="module" src="./../../main.js"></script>
+  <link rel="stylesheet" href="../../style.css">
+
+  <title>Film Book</title>
+</head>
+
+<body>
+  <header-component></header-component>
+  
+  <main class="container">
+    <section class="movie-detail__container">
+      <article class="movie-detail">
+        <div class="movie-detail__background"></div>
+        <img class="movie-detail__image" src="../../images/avengers.jpg" alt="Avengers" width="480" height="730">
```

Comment:
width height - move to css

Reply:
Width height should be in html - especially for img to prevent layout shift -> https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/